packages/core/content-releases/server/src/services/release.ts

```diff
@@ -14,22 +14,41 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
 
     return release;
   },
-  async findMany(query: Record<string, unknown>) {
+  async findMany(query?: Record<string, unknown>) {
     const { results, pagination } = await strapi.entityService.findPage(RELEASE_MODEL_UID, {
-      ...query,
+      // Default query
       populate: {
         actions: {
           // @ts-expect-error TS error on populate, is not considering count
           count: true,
         },
       },
+      // Query request, overrides default
+      ...query,
     });
 
     return {
       data: results,
       pagination,
     };
   },
+  async findOne(id: number, query?: Record<string, unknown>) {
+    const release = await strapi.entityService.findOne(RELEASE_MODEL_UID, id, {
+      // Default query
+      populate: {
+        actions: {
+          // @ts-expect-error TS error on populate, is not considering count
+          count: true,
+        },
+      },
```

Comment:
We are not expecting this endpoint to return the release with its actions? (actions should be sanitized)

Reply:
Returning the actions for a Release is planned in CS-319. This PR would allow Simone to replace mock data on the Release details page. If you agree I think I'd prefer to keep it as is and handle actions on findOne in another PR.

Reply:
Makes sense! Sorry I forgot about that, yes, lets keep it like this so Simone can move forward